apps/sms/js/information.js

```diff
@@ -146,29 +146,34 @@ function showSimInfo(element, iccId) {
     return;
   }
 
-  var info = [];
+  var simInfoElement = element.querySelector('.sim-detail');
   var simId = Settings.getServiceIdByIccId(iccId);
-  var operator = Settings.getOperatorByIccId(iccId);
-  var number = iccManager.getIccById(iccId).iccInfo.msisdn;
-  var data = {};
-  var l10nId;
 
-  info = [operator, number].filter(function(value) {
-    return value;
-  });
+  if (simId === null) {
```

Comment:
I realized that we can never have operator or number if the SIM is unknown, so instead I changed the code like this. Tell me what you think !